<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>764</width>
     <height>334</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>301</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>561</width>
      <height>241</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="childrenTab">
     <attribute name="title">
      <string>Дети</string>
     </attribute>
     <widget class="QTableWidget" name="childrenTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>581</width>
        <height>181</height>
       </rect>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата рождения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Спорт</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="addChildButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeChildButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editChildButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ed</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="trainerTab">
     <attribute name="title">
      <string>Тренера</string>
     </attribute>
     <widget class="QTableWidget" name="trainerTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>581</width>
        <height>181</height>
       </rect>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата рождения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Спорт</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="relations">
     <attribute name="title">
      <string>Расписание</string>
     </attribute>
     <widget class="QTableWidget" name="relationTable">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>581</width>
        <height>181</height>
       </rect>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Тренер</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ребенок</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
